src/components/Events/EventCard.js

```diff
@@ -0,0 +1,134 @@
+import React from 'react'
+import styled from 'styled-components'
+import breakpoint from 'styled-components-breakpoint'
+
+import { Row, Col } from '../grid'
+import StyledLink from '../Common/StyledLink'
+
+import { CardTitle, BodyPrimary, CalendarDay } from '../Typography'
+import remcalc from 'remcalc'
+
+const CalendarPage = styled.div`
+  padding: 24px;
+  border: solid;
+  border-color: ${props => props.theme.colors.border};
+  border-width: thin;
+  width: 108px;
+  height: 108px;
+  text-align: center;
+
+  ${breakpoint('tablet')`
+    width: 143px;
+    height: 143px;
+  `}
```

Comment:
According to James, what I called CalendarPage, is a square with fixed dimensions. If I can't make the component have set widths, how else am I supposed to make it a square with fixed dimensions?

Reply:
Mobile:
![image](https://user-images.githubusercontent.com/7108120/63100224-37ea7000-bfa9-11e9-860e-3730b68de597.png)

Desktop:
![image](https://user-images.githubusercontent.com/7108120/63100244-4173d800-bfa9-11e9-8672-373a4f84c4a3.png)


Column overlay: 
![image](https://user-images.githubusercontent.com/7108120/63100279-52244e00-bfa9-11e9-8a62-40117301e8ee.png)

Reply:
I've had another look at the storybook and how its set up. 

you don't need to set any widths here, particularly because long months such as December or November will start to break if you constrict it. 

Set the width the the 2 columns and use flex to position these elements in the center

Reply:
If the CalendarPage div doesn't have fixed widths and heights (and I meant pre-defined widths, I know they differ by screen width), then I get this
![image](https://user-images.githubusercontent.com/31079643/63104691-5899d200-bf77-11e9-870a-5d18e2209c24.png)

Reply:
and if I add `display: inline-block;` then it's not sqare

Reply:
This is because you're setting the column width to `1` so it takes up 100% of the containers width

Reply:
If I set the width to 2/12, then the column is too narrow for its content, same in 3/12. If I use 4/12 then it fits content but is no longer a square, which according to James is a crucial part of the design. If I'm not to make the squares fixed dimensions and otherwise the DateCard isn't square, then I'm not really sure what to do.